rusk/Cargo.toml

```diff
@@ -37,14 +37,13 @@ thiserror = "1.0"
 anyhow = "1.0"
 rustc_tools_util = "0.2"
 tower = "0.3"
-rand = "0.7"
+rand = "0.8"
 bincode = "1.3.1"
 lazy_static = "1.4"
-rusk-profile = { path = "../rusk-profile" }
-rusk-vm = { git = "https://github.com/dusk-network/rusk-vm", tag = "v0.5.1" }
-canonical = "0.5"
-canonical_derive = "0.5"
-canonical_host = "0.5"
+rusk-profile = {path="../rusk-profile"}
+rusk-vm = {git="https://github.com/dusk-network/rusk-vm", tag="v0.5.1"}
```

Comment:
Any reason why you removed the wrap around the `=` ?

Reply:
Fixed